### DISCUSSION TITLE
Exception thrown at 0x00007FFE891477E5 (ntdll.dll) in puzzle.exe: 0xC0000005: Access violation reading location 0x000001CA00000098.

### DISCUSSION THREAD

**Discussion by sarah-akk** [MuhammadSulaiman001/opengl-lab#18] [Q&A]
this exception was thrown once  debugging  reaches the place where I pointed (glCompileShader)
I searched alot for the reason and didnt find a solution 
![Screenshot 2023-12-02 142551](https://github.com/MuhammadSulaiman001/opengl-lab/assets/130155784/cea29e41-9243-4523-bac2-6462ee14b541)

**Comment by MuhammadSulaiman001** [comment]
Hi @sarah-akk, one of the possible causes for this exception is forgetting to setup opengl loader, for example, if you're using glad, you might be missing (or not calling before gl* functions)
 ```
gladLoadGLLoader((GLADloadproc)glfwGetProcAddress);
``` 
VS debugger might not stop at the line causing the issue... anyway, if this is not your case, I need more context to inspect the issue (unless the code is a homework code 😅).
